toolshed/build_pathfinder_dlls.py

```diff
@@ -7,24 +7,26 @@
 # Input for this script: .txt files generated with:
 # for exe in *.exe; do 7z l $exe > "${exe%.exe}.txt"; done
 
-# The output of this script
-# requires obvious manual edits to remove duplicates and unwanted dlls.
+# The output of this script is expected to be usable as-is.
 
+import collections
 import sys
 
-LIBNAMES_IN_SCOPE_OF_CUDA_BINDINGS_PATH_FINDER = (
+# ATTENTION: Ambiguous shorter names need to appear after matching longer names
+#            (e.g. "cufft" after "cufftw")
```

Comment:
Any way to better protect ourselves here?

Reply:
I didn't want to pack to much into this PR that's not really related to the move. It was more a "minimal while I'm at it" effort to achieve that the output can be used as-is. Previously I had to manually curate, which already led to one accident.

Is it OK to leave more work for later?